server.go

```diff
@@ -643,6 +706,173 @@ func (sp *serverPeer) OnBlock(_ *peer.Peer, msg *wire.MsgBlock, buf []byte) {
 	<-sp.blockProcessed
 }
 
+// OnCmpctBlock is invoked when a peer receives a cmpctblock bitcoin message.
+// It blocks until the bitcoin block has been fully processed.
```

Comment:
Is this comment still accurate? Seems like we are spawning a go routine so it doesn't block...